release notes/v0.9.0.md

```diff
@@ -0,0 +1,111 @@
+k6-browser `v0.9.0` is here 🎉! This release includes:
```

Comment:
I'm a little bit confused with the naming in docs vs in this repo etc. Should this be `k6-browser` or `xk6-browser`? Or even `k6 browser` here?

Reply:
I believe the module name is still `xk6-browser` and since we're referring to the module version, I'd suggest we refer to it by its name.

Reply:
Amended in d686c1cffa45ab8d9ad6f95caaac12750a01b44b